src/sidebar/util/groups.js

```diff
@@ -20,7 +23,48 @@ function combineGroups(userGroups, featuredGroups) {
   featuredGroups.forEach(group => (group.isMember = false));
   userGroups.forEach(group => (group.isMember = true));
 
-  return userGroups.concat(featuredGroups);
+  const groups = userGroups.concat(featuredGroups);
+
+  // Add isSelectable property indicating whether a group can be
+  // annotated in at this particular url.
+  groups.forEach(group => (group.isSelectable = isSelectable(group, uri)));
```

Comment:
I debated making a separate function for this logic entirely tho I'm not sure that's necessary?